include/rgl/api/core.h

```diff
@@ -308,6 +328,18 @@ rgl_entity_destroy(rgl_entity_t entity);
 RGL_API rgl_status_t
 rgl_entity_set_pose(rgl_entity_t entity, const rgl_mat3x4f *local_to_world_tf);
 
+/******************************** SCENE ********************************/
+
+/**
+ * Sets time to the given scene.
+ * Time indicates a specific point in time, relative to a clock's 0 point.
```

Comment:
What does `clock's 0 point` mean? What clock?

Reply:
```suggestion
 * Time indicates a specific point in the simulation timeline when raytrace is performed.
```
Is that cleaner?